src/relay_sdk/workflows.py

```diff
@@ -0,0 +1,38 @@
+"Allows a step to run a workflow"
+import json
+import logging
+from typing import Any
+from collections.abc import Mapping
```

Comment:
This should be imported from `typing` (i.e., added to the line above) instead for compatibility with older Python versions.